packages/select/src/components/select/suggest.tsx

```diff
@@ -73,11 +79,6 @@ export class Suggest<T> extends React.PureComponent<ISuggestProps<T>, ISuggestSt
         return Suggest as new (props: ISuggestProps<T>) => Suggest<T>;
     }
 
-    public state: ISuggestState<T> = {
-        isOpen: (this.props.popoverProps && this.props.popoverProps.isOpen) || false,
```

Comment:
this is missing in new diff

Reply:
test failures due to this i think.

Reply:
Woops, you're most definitely right! Now all the tests succeed!